lib.py

```diff
@@ -672,6 +672,30 @@ def check_blacklist_player_online(client, label_jump):
     else:
         print('[Action] Could not find VIP window')
 
+def check_kill_count(client, monster_name, label_jump, label_skip):
+    result = client.get_windows_by_names(['Quest Tracker'])
+    if client.get_windows_by_names(['Quest Tracker']):
+        tracker = result[0]
+        tracker_text = tracker.recognize_text_content()
+        if monster_name in tracker_text:
+            print('[Action] Monster found')
+            #test with string before if function and limits are correct
+            print('Tracker_Text:', tracker_text)
+            print('Actual Killcount: ', tracker_text[tracker_text.find(monster_name):tracker_text.find('/')])
+            print('Needed Killcount: ', tracker_text[tracker_text.find('/'):tracker_text.find(monster_name)-tracker_text.find('/')])
```

Comment:
this will not work. There is no /
ex: 23/300 is simply 23300
You can try to split by the 300 and get what comes before or check if there is some word that only shows when task is complete.